src/test/java/teammates/test/cases/action/ErrorUserReportLogActionTest.java

```diff
@@ -0,0 +1,70 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.logic.core.AccountsLogic;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.ErrorUserReportLogAction;
+
+/**
+ * SUT: {@link ErrorUserReportLogAction}.
+ */
+public class ErrorUserReportLogActionTest extends BaseActionTest {
+
+    private static final AccountsLogic accountsLogic = AccountsLogic.inst();
+
+    @Override
+    protected String getActionUri() {
+        return Const.ActionURIs.ERROR_FEEDBACK_SUBMIT;
+    }
+
+    @Override
+    protected ErrorUserReportLogAction getAction(String... params) {
+        return (ErrorUserReportLogAction) gaeSimulation.getActionObject(getActionUri(), params);
+    }
+
+    @Override
+    @Test
+    protected void testExecuteAndPostProcess() throws Exception {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+
+        gaeSimulation.loginAsInstructor(instructor1ofCourse1.googleId);
+
+        ______TS("Typical Success Case");
+
+        final String testErrorReportSubject = "Test Error Subject";
+        final String testErrorReportContent = "This is a test user-submitted error report.";
+
+        String[] params = new String[]{
+                Const.ParamsNames.ERROR_FEEDBACK_EMAIL_SUBJECT, testErrorReportSubject,
+                Const.ParamsNames.ERROR_FEEDBACK_EMAIL_CONTENT, testErrorReportContent,
+        };
+
+        ErrorUserReportLogAction action = getAction(params);
+        AjaxResult result = getAjaxResult(action);
+
+        assertEquals(Const.StatusMessages.ERROR_FEEDBACK_SUBMIT_SUCCESS, result.getStatusMessage());
+
+        // getting basic AccountAttributes because ErrorUserReportLogAction only logs this.
+        AccountAttributes instructor1ofCourse1AccountAttributes = accountsLogic
+                .getAccount(instructor1ofCourse1.googleId, false);
+
+        final String expectedLogMessage = "====== USER FEEDBACK ABOUT ERROR ====== \n"
+                + "ACCOUNT DETAILS: " + instructor1ofCourse1AccountAttributes.toString() + "\n"
+                + "SUBJECT: " + testErrorReportSubject + "\n"
+                + "FEEDBACK: " + testErrorReportContent;
+
+        assertEquals(expectedLogMessage,
+                action.getUserErrorReportLogMessage());
+    }
+
+    @Override
+    protected void testAccessControl() throws Exception {
+        // TODO Auto-generated method stub
```

Comment:
Might need to implement this. I'd suggest `InstructorCourseAddActionTest` as an example, but I'm not very familiar with this either.